Okazaki - TERMS - DICT-510

bullenc opened the pull request:
in summary:
- we are fixing typos for a few terms descriptions
- we are updating a few property terms with new NCIt codes/descriptions
- we are adding new properties in this release and thus will need to add new terms entries for these new properties

Commit:
DICT-510_updates

wteouchicago commented on src/gdcdictionary/schemas/_terms.yaml:
```diff
@@ -7700,7 +7718,92 @@ hormonal_replacement_therapy_status:
+  common:
+    description: >-
+      A group of individuals united by a common factor (e.g., geographic location,
+      ethnicity, disease, age, gender)
```
do we need to have a period at the end of the desc? (it's missing in the ticket too.)

bullenc replied:
Great catch, added 👍

wteouchicago commented:
- I didn't see the change for "…described by the Center for Disease Control." for cdc_hiv_risk_factors
- undescended_testis_corrected_age's description and term were already present (not sure if we need to update the ticket). ditto size_extraocular_nodule's term and description.

ebusygina commented:
I see a small difference: the property name is `genome_doubling` in the code, and in the task `genome_doublings`.

Commit:
remove special char

Commit:
minor typo updates

bullenc commented:
> * I didn't see the change for "…described by the Center for Disease Control." for cdc_hiv_risk_factors
> 
>     * undescended_testis_corrected_age's description and term were already present (not sure if we need to update the ticket). ditto size_extraocular_nodule's term and description.

Fixed ticket and yaml, should read 'Centers for Disease Control'.

Removed undescended_testis_corrected_age from ticket, you are correct, a term already existed for it so no need to be added to ticket as a new addition

bullenc commented:
> I see a small difference: the property name is `genome_doubling` in the code, and in the task `genome_doublings`.

fixed ticket 👍 should read `genome_doubling`

ebusygina commented:
undescended_testis_corrected_age: in the code, we added NCIt code to it, probably we need to show it in the task


size_extraocular_nodule: in the code, we added NCIt code to it and didn't add its description and terms. probably, we need to delete its <new description> and <new term> from the task

wwysoc2 commented on src/gdcdictionary/schemas/_terms.yaml:
```diff
@@ -1212,8 +1212,8 @@ columnar_mucosa_present:
-      The text term used to describe a comorbidity disease, which coexists with the
-      patient's malignant disease.
+      The text term used to describe a patient with one or more diseases or conditions,
+      which coexists with the patient's malignant disease at the same time.
```
I seriously have no idea if "coexists" or "coexist" should be used here.

wteouchicago replied:
> I seriously have no idea if "coexists" or "coexist" should be used here.

me too. seems like "one or more" can take the singular or plural, but then the "which" can refer to either, so...

bullenc replied:
Updated to "coexist", I think it flows better and makes more grammatical sense given diseases/conditions are plural

wwysoc2 reviewed (approved):
Looks good, had one comment about grammar.

Commit:
update grammar

bullenc commented:
> undescended_testis_corrected_age: in the code, we added NCIt code to it, probably we need to show it in the task
> 
> size_extraocular_nodule: in the code, we added NCIt code to it and didn't add its description and terms. probably, we need to delete its and from the task

updated ticket to reflect this

stilesj-uchicago reviewed (approved):
Everything looks correct according to ticket DICT-510. My nit is that the terms are not in alphabetical order but that is not required.
